homeassistant/components/duckdns/__init__.py

```diff
@@ -120,7 +120,10 @@ async def interval_listener(now):
                 failed = 0
         finally:
             delay = intervals[failed] if failed < len(intervals) else intervals[-1]
-            remove = async_track_point_in_utc_time(hass, interval_listener, now + delay)
+            # call dt_util.utcnow() again in case time abruptly moves forward
+            remove = async_track_point_in_utc_time(
```

Comment:
Can we use `async_call_later` instead?

Reply:
That would be cleaner. I'll send another PR shortly

Reply:
Actually the `delay` is a timedelta here.

`async_call_later` takes a delay in second as a `float`.

Since we don't have someone who is actually using this, I'm going to leave it as is since refactoring would create risk without having a tester.

Reply:
`timedelta` has a `total_seconds` method:
https://docs.python.org/3/library/datetime.html#datetime.timedelta.total_seconds

Reply:
Adjusted https://github.com/home-assistant/core/pull/39339